scripts/ci/manage-operator.sh

```
@@ -45,11 +45,18 @@ set_tags() {
+      DEBUG_MODE_SUFFIX="-debug"
+    else
+      DEBUG_MODE_SUFFIX=""
+    fi
+
     GIT_COMMIT_ID=$(git --git-dir=${REPOSITORY_PATH}/.git --work-tree=${REPOSITORY_PATH} rev-parse --short HEAD)
     IMAGE_LOC=quay.io/codeready-toolchain/${REPOSITORY_NAME}:${GIT_COMMIT_ID}
     if is_provided_or_paired; then
-        IMAGE_BUILDER=${IMAGE_BUILDER:-"podman"}
-        make -C ${REPOSITORY_PATH} ${IMAGE_BUILDER}-push QUAY_NAMESPACE=${QUAY_NAMESPACE} IMAGE_TAG=${TAGS}
+        make -C ${REPOSITORY_PATH} ${IMAGE_BUILDER}-push${DEBUG_MODE_SUFFIX} QUAY_NAMESPACE=${QUAY_NAMESPACE} IMAGE_TAG=${TAGS}
         IMAGE_LOC=quay.io/${QUAY_NAMESPACE}/${REPOSITORY_NAME}:${TAGS}
```

Shellcheck (0.11.0)

[warning] IMAGE_LOC appears unused. Verify use (or export if used externally).

(SC2034)